WORKING_REMOTE.md

```diff
@@ -0,0 +1,93 @@
+# Working Remote
+
+*The goal of this document is to improve collaboration between people who
+ are not working in the same physical location.*
+
+ **Important: This document is not only for those who are working remotely,
+ but for everyone working at Hyper.
+ Successfully collaboration requires effort from everyone.**
+
+We are all used to speaking with people who are not in the same
+physical place as us. Although we communicate a bitt less effectively over
+phone than when do in person it's usually not a big problem.
+
+However this communication challenge becomes much more apparent to us when
+we're trying to solve complex problems together.
+
+There are many things that makes communicating with remote people more
+difficult. Let's call them communication barriers.
+
+## Communication barriers
+
+To collaborate more effectively we need to remove communication barriers.
+So let us discuss how to solve them and gather their solutions here
+
+### Tips on how to remove collaboration barriers
+
+*This section will focus on how we can remove
+communication barriers by changing our behaviour*
+
+#### Be easily available
+
+Create a [(permanent hangout)](#permanent-hangout) that people can reach
+you on during the day, this makes you equally available as people sitting
+in a physical location at the office.
+
+Posting the bit.ly link in the project group chat at the beginning of the
+day is a great way of inviting people connect. This also reminds them of
+where they can reach you for a quick talk.
+
+#### Meetings
+
+(Work in progress).
+
+When in a meeting think about where you sit relative to microphones and
+cameras. Sit close to the microphone and camera. Properly hearing and seeing
+all the people in a meeting makes wonders for good communication.
+
+Remove objects that might cause loud noises thought the microphone:
+
+*   Metal bracelets creates a painful noise in the microphone
+when scratched against a macbook or the table for example.
+
+Also consider having an all virtual meeting. When everybody have their own
+camera and microphone communication is often much better.
+This is specially true when you are discussing something visual like
+design changes over a shared screen.
+
+(more must be written about this.)
+
+### Tools that remove collaboration barriers
+
+*This section will focus on tools we can use to remove
+communication barriers*
+
+*   Effective use of Hangouts
+*   Permanent Hangout, with bit.ly link. [(read more bellow)](#permanent-hangout)
+*   Good quality microphones (and enough of them)
+*   Good quality cameras (and enough of them)
+
+#### Permanent-Hangout
+
+To make a permanent hangout Follow these steps:
+
+1.  Schedule a meeting with your gmail account
+(preferably your private gmail account*). Be sure to create a hangout session
+that belongs to this meeting (should be on by default).
```

Comment:
I think the meeting has to be in the future in order for the Hangout to not expire, so best make a meeting that happens in 40 years or something.